py/get_tiles.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
```

Comment:
yeah i left it in there for future use....otherwise I will forget where it is stashed :-)